ci/log_analyze_setup.sh

```diff
@@ -31,19 +31,35 @@ fi
 # The new complex password we use for jenkins test
 export SNOWFLAKE_TEST_PASSWORD_NEW="ThisIsRandomPassword123!"
 
-LOG_PROPERTY_FILE_DOCKER=$(cd "$(dirname "${BASH_SOURCE[0]}")/.."; pwd)/src/test/resources/logging.properties
+LOG_PROPERTY_FILE=$(cd "$(dirname "${BASH_SOURCE[0]}")/.."; pwd)/src/test/resources/logging.properties
 
 export CLIENT_DRIVER_NAME=JDBC
 
 function setup_log_env() {
-    sed -i "s|^java.util.logging.FileHandler.pattern.*|java.util.logging.FileHandler.pattern = $CLIENT_LOG_FILE_PATH_DOCKER|" ${LOG_PROPERTY_FILE_DOCKER}
+    CLIENT_LOG_DIR_PATH=$LOCAL_CLIENT_LOG_DIR_PATH
```

Comment:
we can move the 3 assignements to the else branch of the if below?

Reply:
Yes, done.